Tasks/AzurePowerShellV3/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 3,
         "Minor": 1,
-        "Patch": 1
+        "Patch": 2
```

Comment:
Other tasks such as AzureFileCopy will also need to be updated as they use the same initialize function.